### ISSUE TITLE
MinkContext::assertHomePage() failing

### ISSUE THREAD

**Issue by alfaguru** [Behat/MinkExtension#163]
In my testing I encountered an odd failure whereby when although correctly on the homepage assertHomePage() was returning an exception:
 Current page is "", but "/" expected. (Behat\Mink\Exception\ExpectationException)

There seems to be some inconsistency between URL handling in MinkContext and WebAssert, though I have not been able to fully isolate the cause: it seems WebAssert is stripping the '/' and MinkContext is forcing it to be present. Not sure which is at fault.

Currently I am working round this by using assertUrlRegExp('#^/?$#') instead. At least this will work with either version of the path.

**Comment by cmcnamara87** [comment]
+1

**Comment by bpawan** [comment]
:+1:

**Comment by bpawan** [comment]
+1

**Comment by aik099** [comment]
@alfaguru , is PR coming?

**Comment by alfaguru** [comment]
I haven't patched the source. I just worked round the problem as I mentioned. I didn't find a possible fix and it's not clear to me which of the two components is responsible.

**Comment by aik099** [comment]
Can you please post Gherkin scenario used.

**Comment by aik099** [comment]
From what I've found there are 2 steps that might be of use:

* `I should be on "page-name"` - checks that we're on given page
* `I should be on the homepage` - checks that we're on homepage

I guess you're using 1st one, but should be using 2nd one.

**Comment by aik099** [comment]
I've tried the following scenario with Goutte Mink's driver and all passed:
```gherkin
Scenario: Going to homepage
	When I go to "/"
	Then I should be on the homepage
```

**Comment by aik099** [comment]
I think, that I've found why you're expiriencing that problem, but I don't. There was a PR https://github.com/minkphp/Mink/pull/616 in Mink, that fixes case, when homepage url was detected incorrectly. The fix is available since Mink 1.6.1 release. I guess you're not using that release.

**Comment by alfaguru** [comment]
Thanks, that sounds promising. I had some issues with Behat dependencies so haven't updated in a while. Will take another look.
